gtest/src/program.rs

```diff
@@ -206,11 +207,12 @@ impl<'a> Program<'a> {
             .expect("Unable to get root directory of the project")
             .join(path)
             .clean();
-
+        let is_meta = path.to_str().unwrap_or_default().ends_with("meta.wasm");
```

Comment:
such functionality will be resolved in #1096

Reply:
no need in this changes within PR of fallibility of meta_state fn

Reply:
Removed.